### DISCUSSION TITLE
Ability to programmatically change warmups

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#115] [Ideas]
Similar how you can programmatically change reps/weights of the regular sets now with `reps`, `weights` - do the same for warmups, so that you can change them with `warmupReps`, `warmupWeights`.

**Comment by Max-Tayne** [comment]
This would be really cool.

On a related note, I have been experimenting with an RPE-based progress/update logic, and I'd love to be able to **take RPE user input from the warmup sets** to determine the regular working set weights (and the increment/number of sets of the warmups). 

I've found that the optimal working weight for my target RPE can vary greatly day to day, and this method of handling it really helps me not have to type and click as much during workouts.
